index.js

```diff
@@ -76,13 +110,20 @@ const after_createDeploymentArtifacts = bluebird.coroutine(
 
     yield cljsLambdaBuild(serverless, opts);
 
+    yield applyZipExclude(serverless, opts);
+
     serverless.cli.log(`Returning artifact path ${serverless.service.__cljsArtifact}`);
     return serverless.service.__cljsArtifact;
   });
 
+const defaultExclude = ['node_modules/serverless-cljs-plugin'];
```

Comment:
Would you mind terribly calling this `DEFAULT_EXCLUDE` and moving it to the top of the file?

Does this need a glob, or does it work as is?

Reply:
- No problem I will do it.
 - Yep it works this way, `minimatch` matches and no dir is created in the zip file